ROADMAP.md

```diff
@@ -2,20 +2,30 @@
 
 This document defines a high level roadmap for Rook development. The dates below are subject to change but should give a general idea of what we are planning. We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 0.6 (September 2017)
+## Rook 0.7
 
- - Ceph Block storage declared Beta on K8S
- - Kubernetes 1.8 Support
- - Rook volume plugin support in K8S
- - Rook agent on client nodes
- - Rook is upgradeable
- - Support for local storage (and arbitrary volumes) underneath OSD pods
- - Support for public cloud environments
- - Improved test matrix and more test coverage
- - CRDs for object storage and file system
+- Ceph Block Storage (Cluster and Pool CRDs) declared beta
+- Mon reliability (restarts, failing over too fast, ip changes, etc.)
+- Durability of state (local storage support, config is regenerat-able)
+- Run everywhere (rook-agent uses ceph-fuse, nbd-rbd / tcmu runner)
+- adding / removing nodes and disk drives (lifecycle issues, failures, etc.)
```

Comment:
I think it looks better and more "streamlined" if the first character of the sentence is upper case like for the other points in the list(s).